src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -329,9 +331,18 @@ private void visitScript(NodeTraversal t, Node script) {
 
       String moduleName = inputToModuleName(t.getInput());
 
-      // Rename vars to not conflict in global scope.
-      NodeTraversal.traverseEs6(compiler, script, new SuffixVarsCallback(
-          moduleName));
+      boolean hasExports = !(moduleExportRefs.isEmpty() && exportRefs.isEmpty());
+
+      // Rename vars to not conflict in global scope - but only if the script exports something.
```

Comment:
Yes - they would both exist in the global scope. Unless, it's actually imported by something else: see #1476